site/documentation/content/user-guide/device-registry.md

```diff
@@ -314,23 +314,21 @@ ETag: becc93d7-ab0f-48ec-ad26-debdf339cbf4x
   "enabled": true,
   "secrets": [
     {
-      "pwd-hash": "$2a$10$uc.qVDwXeDRE1DWa1sM9iOaY9wuevjfALGMtXmHKP.SJDEqg0q7M6",
-      "hash-function": "bcrypt"
+      "id": "349556ea-4902-47c7-beb0-1009ab693fb4"
     }
   ]
 }]
 ~~~
-    
+The above result does not contain `pwd_hash` and `hash_function` of the secret, as these values are confidential.
+
 The following commands add some `hashed-password` credentials for device `4720` using authentication identifier `sensor20`:
```

Comment:
does this actually *add* credentials or does it *replace* all existing credentials? IMHO it is the latter ...

Reply:
At this point of the guide there are no credentials registered for device `4720`, so it actually `add` some. I changed the wording to be more clear though.